reviews/templates/documents_list.html

```diff
@@ -0,0 +1,41 @@
+    
+{% load static %}
+{% load proposal_filters %}
+{% load fetc_filters %}
+{% load i18n %}
+{% load compare_tags %}
+{% load documents_list %}
+
+{% for header, items in headers_items.items %}
+
+    <h5>
+       {{header}}
+    </h5>
+
+    <ul>
+    {% for name, file, obj in items %}
+        <li>
+            <a href="{{ file.url }}" target="_blank">
+                {{name}}
+            </a>
+            {% if file.field.name != "pdf" %}
```

Comment:
Why are we filtering PDF files here?

Reply:
Oh it's the proposal PDF! I read it as filterring pdf's in general. This makes more sense. 
(Maybe we should add a comment what it's actually filtering)